components/shared/DragDrop.vue

```diff
@@ -37,10 +37,13 @@
           class="flex items-center"
         >
           <div class="flex flex-col justify-between">
-            <KIcon
-              name="i-mdi:check-circle-outline"
-              class="check-icon text-k-green"
-            />
+            <div>
+              <KIcon
+                name="i-mdi:check-circle-outline"
+                class="check-icon text-k-green"
+                size="3x"
+              />
+            </div>
```

Comment:
It doesn't need to add an extra div, but only need to add style `items-center`

```<div class="flex flex-col justify-between items-center">```

```suggestion
              <KIcon
                name="i-mdi:check-circle-outline"
                class="check-icon text-k-green"
                size="3x"
              />
```



![image](https://github.com/user-attachments/assets/c6b74f28-4774-4919-9ec2-07aad76d964d)